.jenkins/Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 
 local_lib=""
 
-changeAuthor = env.ghprbPullAuthorLogin ?: CHANGE_AUTHOR
+changeAuthor = getGroup() ?: CHANGE_AUTHOR
```

Comment:
you now we already have  the `util.getGroup(getProject())` method, right?

Reply:
Yeah I know and I copied the code from there :D

The issue here is that I want to test the library dynamically, so don't want to load the library for the whole pipeline, else it will mess with the current testing in this Jenkinsfile ;)